src/umpire/strategy/QuickPool.hpp

```diff
@@ -99,6 +99,12 @@ class QuickPool : public AllocationStrategy, private mixins::AlignedAllocation {
     {
     }
 
+    ~pool_allocator()
+    {
+      if (pool != nullptr)
+        delete(pool);
```

Comment:
The only question I had here was whether we can use RAII instead of having to perform a `new/delete` pair?  I don't believe that pointer is required.

Reply:
Good idea. I could try play around with the pool pointer and see if it can be left a normal variable, not a pointer.

Reply:
Ok - I took out the pointer and it seems to pass all the AddressSanitizer/LeakSanitizer tests.. I can do another push and y'all can check it out. I assume it is best to remove the pointer?

Reply:
Yes, please.